src/SortChain.php

```diff
@@ -42,6 +48,10 @@ public function keys($collection)
 
     public function __invoke($a, $b)
     {
+        if (empty($this->callables)) {
+            return static::$nullOperatorEquality;
```

Comment:
Haven't tried it but theoretically repeatable with a list of similar elements. I could allow fallthrough to the "return 0" but there's no telling if this is just a single pair of identical elements or an entire collection that shouldn't be reordered.